### PR TITLE
[WIP: not for 0.8.0] Modified Content

### PR DESCRIPTION
**Pull request by j-c-berger**
Signed-off-by: Jacob Berger <jacob.berger@ibm.com>

For issue 1756: https://github.com/eclipse/codewind/issues/1756

I simply modified content to fit our new restructure

@micgibso or @sishida, please review and let me know what other corrections must be made or merge it if it's ready. Thanks!

### CODE REVIEW

File: docs/_documentations/vsc-codechange.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
-title: "Getting started with Codewind for VS Code"
-description: "Getting started with Codewind for VS Code"
+title: "Making a code change"
+description: "Making a code change"
```
Comment (sishida) [inline docs/_documentations/vsc-codechange.md]: If all the IDEs will have a file of the same title, I'm thinking we should add the name of the IDE as part of the title. Adding the name could also help orient readers to what's going on in this file if they happen to see this page out of context from the rest of the table of contents.

Maybe something like..."Making a code change with VS Code"?

File: docs/_documentations/vsc-codechange.md [hunk 2]
```diff
@@ -13,31 +13,31 @@ parent: mdt-vsc-overview
 <br/>
 Codewind automatically builds and redeploys your application whenever you make a code change and save the file.
 
-To see this in action, we can make a change to the getting started example.
+To see this feature in action, make a change to the getting started example.
 
-First step is to edit the index.html file
+Edit the index.html file
```
Comment (sishida) [inline docs/_documentations/vsc-codechange.md]: End the sentence with a colon or period.

File: docs/_documentations/vsc-codechange.md [hunk 3]
```diff
@@ -13,31 +13,31 @@ parent: mdt-vsc-overview
...
 
 ![](dist/images/vsc-codechange.png){:width="800px"}
 
 and navigate to the bottom of the file to find the lines
 
 ![](dist/images/vsc-codeline.png)
 
-Change the heading from *Congratulations* to be *I did this* 
+Change the heading from *Congratulations* to *I did this*
```
Comment (sishida) [inline docs/_documentations/vsc-codechange.md]: This sentence could end with a period.

**Comment by j-c-berger** [comment]
@sishida, thank you for the review. I applied the changes. You or @micgibso can review and/or merge, unless there's more to do!

**Comment by j-c-berger** [comment]
@micgibso, new commit is ready for review and hopefully merging. I already pinged Sarah about it. Thanks!

**Pull request merged**